python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -236,6 +236,13 @@ def bypass_deepMergedCoaddId_bits(self, *args, **kwargs):
     def bypass_deepMergedCoaddId(self, datasetType, pythonType, location, dataId):
         return self._computeCoaddExposureId(dataId, False)
 
+    def bypass_dcrMergedCoaddId_bits(self, *args, **kwargs):
+        """The number of bits used up for patch ID bits"""
+        return 64 - self._nbit_id
```

Comment:
Yes, that would be much better. I'll correct all the bypass functions to call the deepCoadd version.